fhir-validation/src/main/java/com/ibm/watson/health/fhir/validation/FHIRValidator.java

```diff
@@ -117,8 +120,7 @@ private FHIRPathResourceNode getResource(Class<?> type, FHIRPathNode node) {
             return null;
         }
 
-        private void validate(Class<?> type) {
-            Set<Constraint> constraints = ModelSupport.getConstraints(type);
+        private void validate(Class<?> type, Collection<Constraint> constraints) {
```

Comment:
what is this method validating?  maybe add javadoc?

Reply:
I guess its validating the the nodes resolved from the path returned by getPath() conforms to the passed constraints...just wondering if we can make it more obvious from either signature or javadoc

Reply:
and yes, i know its private method, was just trying to make sure i understand what it was doing...

Reply:
Will add javadoc